resources/ext/ezbake.conf

```diff
@@ -2,71 +2,21 @@ ezbake: {
-               install: ["echo \\\"os-settings: {\\\"                         > %{buildroot}%{_sysconfdir}/%{realname}/conf.d/os-settings.conf",
-                         "echo \\\"    ruby-load-path: [/opt/puppetlabs/agent/lib/ruby/vendor_ruby]\\\" >> %{buildroot}%{_sysconfdir}/%{realname}/conf.d/os-settings.conf",
-                         "echo \\\"}\\\"                                      >> %{buildroot}%{_sysconfdir}/%{realname}/conf.d/os-settings.conf" ],
-               # This is terrible, but we need write access to puppet's
-               # var/conf dirs, so we need to add ourselves to the group.
-               # Then we need to chmod some dirs until the Puppet packaging
-               # is changed to allow group write.
-               # TODO: The Puppet agent package is currently missing some
-               # things. We should clean this up if possible once puppet-agent
-               # packages are settled. See SERVER-333.
-               postinst: ["install --directory --group=puppet --owner=puppet /var/lib/puppet/jruby-gems",
-                          "usermod -d /opt/puppetlabs/agent/cache puppet",
-                          "chown puppet:puppet /opt/puppetlabs/agent/cache",
-                          "chmod 775 /opt/puppetlabs/agent/cache",
-                          "install --directory /opt/puppetlabs/agent/cache/reports",
-                          "chown puppet:puppet /opt/puppetlabs/agent/cache/reports",
-                          "chmod 775 /opt/puppetlabs/agent/cache/reports",
-                          "echo \\\"[master]\\\"                               > /etc/puppetlabs/agent/puppet.conf"
-                          "echo \\\"vardir = /opt/puppetlabs/agent/cache\\\"   >> /etc/puppetlabs/agent/puppet.conf",
-                          "echo \\\"logdir = /var/log/puppetlabs\\\"           >> /etc/puppetlabs/agent/puppet.conf",
-                          "echo \\\"rundir = /var/run/puppetlabs\\\"           >> /etc/puppetlabs/agent/puppet.conf",
-                          "chown puppet:puppet /etc/puppetlabs/agent/",
-                          "chmod 775 /etc/puppetlabs/agent",
-                          "install --directory /etc/puppetlabs/agent/ssl",
-                          "chown -R puppet:puppet /etc/puppetlabs/agent/ssl",
-                          "chmod -R 770 /etc/puppetlabs/agent/ssl",
-                          "chown -R puppet:puppet /var/log/puppetlabs/",
-                          "chmod -R 770 /var/log/puppetlabs"]
+               postinst: [
+                 "/opt/puppetlabs/puppet/bin/puppet config set --section master vardir  /opt/puppetlabs/server/data/puppetserver",
+                 "/opt/puppetlabs/puppet/bin/puppet config set --section master codedir /etc/puppetlabs/code",
+                 "usermod --home /opt/puppetlabs/server/data/puppetserver puppet",
+                 "install --directory --owner=puppet --group=puppet --mode=775 /opt/puppetlabs/server/data"
```

Comment:
> Should this go all the way down to /opt/puppetlabs/server/data/puppetserver?

Yes, but I plan to take care of that in a subsequent ticket since it wasn't technically required to get smoke tests passing.

> Also, should there be a step here to create the subordinate "jruby-gems" directory?

Maybe?  Our smoke tests are passing without it because the `gem` subcommand should automatically create the parent directory upon an `install` action provided the parent directory is writable, which it is.